src/plugins/analysis/kernel_config/view/kernel_config.html

```diff
@@ -7,6 +7,30 @@
             <td>Kernel Config</td>
             <td><pre class="border rounded p-2 bg-light" style="white-space: pre-wrap; height: 450px; overflow: scroll;"><code>{{ firmware.processed_analysis[selected_analysis]['kernel_config'] | safe }}</code></pre></td>
         </tr>
+        <tr>
```

Comment:
It would be good to add an `{% if ... %}` check here to check if there are results (or else there might be a `KeyError`)